internal/common/conversion/type_conversion.go

```diff
@@ -3,8 +3,18 @@ package conversion
 import (
 	"strings"
 	"time"
+
+	"github.com/hashicorp/terraform-plugin-framework/types"
 )
 
+func SafeValue[T any](v *T) T {
+	if v != nil {
+		return *v
+	}
+	emptyValue := new(T)
+	return *emptyValue
```

Comment:
```suggestion
	var emptyValue T
	return emptyValue
```

that is good, curious if this alternative also works

Reply:
Good idea. Changed in 4b16f455ea32f9f74f93c78d8fcbc9c889589139